docker-compose.yml

```diff
@@ -7,9 +7,7 @@ services:
       - MYSQL_USER=remo
       - MYSQL_PASSWORD=remo
       - MYSQL_ROOT_PASSWORD=root
-    entrypoint:
-      - docker-entrypoint.sh
-      - --max_allowed_packet=2M
+    command: mysqld --max_allowed_packet=64M
```

Comment:
During the initialize on my system there a crash for the huge amount of importing from json.
With this parameter I fixed that problem.
For the docker-entrypoint.sh I removed because that file doesn't exist and I get error on getting the project working.